src/string_bytes.cc

```diff
@@ -150,62 +150,83 @@ static const int unbase64_table[] =
     -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1,
     -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1
   };
-#define unbase64(x) unbase64_table[(uint8_t)(x)]
+#define unbase64(x)                                                           \
+    static_cast<uint8_t>(unbase64_table[static_cast<uint8_t>(x)])
```

Comment:
> Is four spaces here, okay?

I don't think we really have a convention for that but I'll change it to two spaces before landing.